runner/execer/os/os_execer.go

```diff
@@ -82,8 +83,12 @@ type osProcess struct {
 
 // Periodically check to make sure memory constraints are respected.
 func (p *osProcess) monitorMem(memCap execer.Memory, stat stats.StatsReceiver) {
-	memTicker := time.NewTicker(100 * time.Millisecond)
+	prevThreshold := 0.0   // The last reported (usage / mem_cap)
+	reportThreshold := 0.0 // Start reporting once we hit this utilization threshold.
```

Comment:
1. shouldn't this be higher (to reduce log messages) - (60%, 70% or 80%?) - also if we increase this threshold we should probably decrease the increment to something like 5%...
2. do we want to report if/when usage goes back under the threshold?